server/config.py

```
@@ -0,0 +1,221 @@
+        Get config option by section and key name.
+        
+        Args:
+            section: Configuration section name
+            key: Configuration key name
+            
+        Returns:
+            Configuration value or None if not found
+        """
+        try:
+            return self.config.get(section, key)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)